src/main/java/oop/entertainment/Entertainment.java

```diff
@@ -0,0 +1,38 @@
+package oop.entertainment;
+/*
+ * Creates a class "Entertainment" - no instance can be created of it
+ */
+public abstract class Entertainment{
+    /*
+     * A class "title" that has specific attributes - all entertainment has a title
+     */
+    private String title;
+    /*
+     * A getter method for title - accesses the private specific variable title
+     */
+    public String getTitle(){
+        return this.title;
+    }
+    /*
+     * A setter method for title - creates new instane of the variable title
+     */
+    public void setTitle(String title){
+        this.title = title;
+    }
+    /*
+     * A class "Pasttime" that has specific attributes - all entertainment passes time
+     */
+    private String pasttime;
```

Comment:
this is a fine location for this data member - generally, we want to group the members at the top of the class file and then run our setters / getters below for readability, but this is acceptable style as well.